style.css

```diff
@@ -1,4 +1,312 @@
+  padding: 46px 108px 55px 97px;
+  width: 259px;
+}
+
+.language__img {
+  height: 25px;
+  width: 31px;
+}
+
+.language__arrow {
+  height: 8px;
+  margin: 10px;
+  width: 16px;
+}
+
+.header-menu {
+  background-color: #f2f2f2;
+  height: 92px;
+  margin-left: 1348px;
+  position: absolute;
+  top: 0;
+  width: 92px;
+}
+
+.header-menu__line::before {
+  content: '';
+  transform: translateY(-11px);
+}
+
+.header-menu__line, 
+.header-menu__line::before {
+  background: #000;
+  display: block;
+  height: 4px;
+  left: 50%;
+  margin-left:-18px;
+  position: absolute;
+  top: 53%;
+  width: 36.5px;
+}
+
+.collection {
+  font-size: 0;
+  padding-top: 77px;
+  position: relative;
+  width: 100%;
+}
+
+.collection__img {
+  display: inline-block;
+  float: left;
+  height: 562px;
+  position: relative;
+  width: 549px;
+  margin-left: 186px;
+  margin-right: 53px;
+}
+
+.collection__title {
+  display: inline-block;
+  font-size: 80px;
+  font-style: normal;
+  font-weight: bold;
+  height: 164px;
+  line-height: 82px;
+  margin-bottom: 3px;
+  margin-top: 102px;
+  position: relative;
+  width: 475px;
+}
+
+.collection__description {
+  display: inline-block;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  height: 127px;
+  margin-bottom: 37px; 
+  position: relative;
+  width: 467px;
+}
+
+.collection__btn {
+  background-color: #fff;
+  border: 1.5px solid #000;
+  font-family: Montserrat;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 500;
+  display: inline-block;
+  height: 44px;
+  margin-right: 53px;
+  position: relative;
+  text-transform: uppercase;
+  transform: translateX(-1px);
+  width: 203px;
+}
+
+.range {
+  display: inline-block;
+  position: relative;
+}
+
+.range__link {
+  color: #000;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 500;
+  display: inline-block;
+  margin-right: 17px;
+  position: relative;
+  text-decoration: none;
+  text-transform: uppercase;
+  width: 172px;
+}
+
+.rangle__link-arrow {
+  display: inline-block;
+}
+
+.socials-list {
+  display: flex;
+  justify-content: space-between;
+  height: 43px;
+  margin-left: 52px;
+  margin-top: 155px;
+  padding: 0;
+  width: 160px;
+ }
+
+.socials__item-pic {
+  height: 43px;
+  margin-right: 8px;
+  width: 43px;
+}
+
+.scroll {
+  display: flex;
+  height: 46px;
+  margin-bottom: 18px;
+  margin-left: 707px;
+  transform: translateY(-42px);
+  width: 27px;
+}
+
+.scroll__btn {
+  background-color: white;
+  box-sizing: border-box;
+  border-radius: 15px;
+  cursor: pointer;
+  position: relative;
+  z-index: 1;
+}
+
+.scroll__wheel { 
+  background: #000;
+  border-radius: 19px;
+  height: 8px;
+  display: inline-block;
+  margin-top: 10px;
+  width: 3px;
+  transform: translateX(-15px);
+  z-index: 2;
+}
+
+@media (min-width: 375px) and (max-width: 480px) {
+
+  * {
+  	box-sizing: border-box;
+  }
+
+  html,
+  body {
+  	margin: 0 auto;
+  	max-width: 480px;
+  	min-width: 375px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5658198)